### PR TITLE
Add Yamaha USB MIDI driver v1.5.0

### PR DESCRIPTION
**Pull request by Flopsey**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
I am not sure what the issue is here -- the CI tests passed on the initial run, and I am not near my testing machine at the moment.

@bevanjkay mind having a look?

**Comment by bevanjkay** [comment]
@miccal Looks like it worked this time around

**Comment by miccal** [comment]
Weird, but ok!

**Comment by miccal** [comment]
Thanks @Flopsey.

File: Casks/yamaha-usb-midi-driver.rb [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-  name "yamaha-usb-midi-driver"
+  name "Yamaha USB MIDI driver"
   desc "MIDI driver for Yamaha instruments"
   homepage "https://usa.yamaha.com/support/updates/usb_midi_driver_for_mac.html"
 
   livecheck do
     url :homepage
     strategy :page_match
-    regex(%r{<h1>.*?V(\d+(?:\.\d+)*(?:-\d+)?).*?</h1>}i)
+    regex(/USB-MIDI\sDriver\sV?(\d+(?:\.\d+)+)\sfor\sMac/i)
```
Comment (Flopsey) [inline Casks/yamaha-usb-midi-driver.rb:13]: I think the `(?:-\d+)?` part is important too: There are older versions with a dash after the patch number, like [1.3.2-2](https://web.archive.org/web/20210320203234/https://usa.yamaha.com/support/updates/usb_midi_driver_for_mac.html). It's quite possible they will use that notation again for future releases.
Forgot to mention that before.

Comment (miccal) [reply Casks/yamaha-usb-midi-driver.rb:13]: No problem, we can fix that in the next update :)

**Pull request merged**